exdb/tests.py

```diff
@@ -197,3 +202,42 @@ def test_load(self):
         self.driver.get(CustomRunner.live_server_url)
         self.assertEquals(self.driver.find_element(By.XPATH, '//h1').text, _('Welcome'))
 
+class PendingApprovalQueueBrowserTest(DefaultLiveServerTestCase):
+    def test_load(self):
+        self.driver.get(CustomRunner.live_server_url + '/pending')
+        self.assertEquals(self.driver.find_element(By.XPATH, '//h1').text, _('Experiences Pending Approval'))
+
+
+"""*******************Integration Tests**********************"""
+
+class StandardTestCase(TestCase):
+    def setUp(self):
+        self.test_user = User.objects.create(username="Test User")
+        self.test_user.set_password('a')
+        self.test_user.save()
```

Comment:
These three lines could be condensed into one using the [create_user()](https://docs.djangoproject.com/es/1.9/ref/contrib/auth/#django.contrib.auth.models.UserManager.create_user) function:
```python
self.test_user = get_user_model().objects.create_user('test_user', 'test@user.com', 'a')
```